src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -412,7 +412,7 @@ public int Main(string[] args)
                     }
 
                     var command = new DependencyListCommand(options, _environment.RuntimeFramework);
-                    return command.Execute();
+                    return command.Execute(_hostServices);
```

Comment:
I suggest we follow one consistent pattern in this file. That is, for class `XYZCommand`, we pass environment info/services it needs to its ctor and `XYZCommand.Execute()` should take no parameter.